operator/bundle_helpers/README.md

```diff
@@ -43,12 +41,14 @@ chmod +x pip_find_builddeps.py
 pip-compile requirements-build.in --allow-unsafe --generate-hashes
 ```
 
+4. Exit the container and commit the changes.
+
 For more information, consult the [Cachi2 docs](https://github.com/containerbuildsystem/cachi2/blob/main/docs/pip.md#building-from-source).
 
 ### What does each requirements file do?
 
 * `requirements.in`: List of project dependencies.
-* `requirements-gha.txt`: Temporary list of project dependencies as required by the build process on GHA and locally. This will be deleted after ROX-26860.
+* `requirements-gha.txt`: Temporary list of project dependencies as required by the build process on GHA and locally. This workaround exists due to a different Python version. Any changes in this or the `requirements.in` file must be synced manually to the other. This will be deleted after ROX-26860.
```

Comment:
```suggestion
* `requirements-gha.txt`: List of project dependencies as required by the build process on GHA and locally. This file exists as a workaround due to a different Python version in this context. Any changes in this or the `requirements.in` file should be synced manually to `requirements-gha.txt`. This file will be deleted after ROX-26860.
```

Reply:
done in https://github.com/stackrox/stackrox/pull/14257/commits/817a312c46b790f483f0e147f223493ec2872f67